TaskManager/TaskManagerTests/MockDataStorage.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MockDataStorage.swift
+//  TaskManagerTests
+//
+//  Created by user on 13.10.2022.
+//
+
+import XCTest
+@testable import TaskManager
+import CoreData
+
+class MockDataStorage: HomeDataStorage {
+
+    var isCreateTaskCalled = false
+    var isDeleteTaskCalled = false
+    var isUpdateTaskCalled = false
+    var taskManagerModeltask = [TaskManager.ModelTask]()
+    var isSaveTaskCalled = false
+    var isRollbackTaskCalled = false
+    var isDeleteAllTaskCalled = false
+  
```

Comment:
lets create a function that will reset the values for this stored vars

Reply:
Didn't see a fix for this

Reply:
its ok, there is no need, because he has a different approach to reset them